src/impls.rs

```diff
@@ -1093,6 +1093,43 @@ impl<T: Config> Pallet<T> {
 		// }
 		Ok(())
 	}
+
+	// Updates the stake amount after retrieving the latest token price from oracle.
+	pub(super) fn _update_stake_amount() -> DispatchResult {
```

Comment:
I guess, we should not use underscore in the beginning of the function's name. Can we please remove it if you agree on this?

Reply:
There is a clash with the dispatchable function of the same name unfortunately.

I am not sure what the convention is in Rust, maybe do_, but _ is used in the Tellor smart contracts for internal functions, hence the reason why I used it for those that clash within impls.rs.

Any suggestions?

Reply:
As per my understanding, in Rust we use underscore when the function is not being used anywhere.
Yes, it'd be good if we use do_ just to make our code align with Rust standards.

Reply:
Will add a final commit 👍